src/pages/public_registration/preview_file/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './PreviewFile';
```

Comment:
Hvorfor index-fil?

Reply:
syns det er så stygt med `import PreviewFile from './preview_file/PreviewFile';` :p

Reply:
fordi `PreviewFile` står to ganger? vi hadde faktisk den diskusjonen der innad i Variant for et par uker siden. Mange delte meninger. Jeg synes det er tungvint med masse index-filer overalt for da er det vanskeligere å søke etter spesifikke filer innad i vscode f.eks. Da vises bare "index.ts" "index.ts" osv... isteden for "PreviewFile.tsx" "PdfFile.tsx"

Reply:
Ja, men enig i at mange `index.ts` er ganske æsj... Egentlig ville jeg bare hatt en måte å eksportere bare `PreviewFile` fra denne folderen, for alle andre filer er egentlig bare helpers for å previewe fil, men bør bli brukt mellom PreviewFile, men det er ikke mulig så litt av poenget forsvinner allerede der.

Er egentlig ikke kjempefan av default export/import heller, for når filer endrer navn blir det ofte en mismatch mellom hva komponenten heter når den importeres og faktisk filnavn, som kan være forvirrende.

dropp `index.ts` og bruke named export/import i stedet?

Reply:
Ja, egentlig enig i at default export er dumt også. Kan jo kjøre på med named fra nå av da 👍🏼